src/Subscription/Subscription/Az.Subscription.psd1

```diff
@@ -72,7 +72,7 @@ NestedModules = @('Microsoft.Azure.PowerShell.Cmdlets.Subscription.dll')
 FunctionsToExport = @()
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = 'New-AzSubscription', 'Update-AzSubscription'
+CmdletsToExport = 'New-AzSubscription','Update-AzSubscription','New-AzSubscriptionAlias','Get-AzSubscriptionAlias','Remove-AzSubscriptionAlias'
```

Comment:
Please submit a cmdlet review here https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues?q=is%3Aissue+Subscription for new cmdlet

Reply:
Cmdlets already approved, please check here - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/691